### ISSUE TITLE
TypeError: this is not a typed array.

### ISSUE THREAD

**Issue by klubi** [postmanlabs/newman-docker#67]
```
/usr/lib/node_modules/newman/lib/reporters/html/index.js:79
                        reducedExecution.response.body = Buffer.from(reducedExecution.response.stream).toString();
                                                                ^

TypeError: this is not a typed array.
    at Function.from (native)
    at /usr/lib/node_modules/newman/lib/reporters/html/index.js:79:65
    at /usr/lib/node_modules/newman/node_modules/lodash/lodash.js:13767:16
    at arrayEach (/usr/lib/node_modules/newman/node_modules/lodash/lodash.js:537:11)
    at Function.transform (/usr/lib/node_modules/newman/node_modules/lodash/lodash.js:13766:43)
    at EventEmitter.<anonymous> (/usr/lib/node_modules/newman/lib/reporters/html/index.js:60:28)
    at EventEmitter.emit (/usr/lib/node_modules/newman/node_modules/eventemitter3/index.js:151:33)
    at _.assignIn.done (/usr/lib/node_modules/newman/lib/run/index.js:221:29)
    at /usr/lib/node_modules/newman/node_modules/postman-runtime/lib/backpack/index.js:56:34
    at PostmanCollectionRun._.assign._process (/usr/lib/node_modules/newman/node_modules/postman-runtime/lib/runner/run.js:133:20)
```

I get above error only when I run in docker (latest postman/newman_alpine33)
Looks like it's related to https://github.com/postmanlabs/newman/issues/1102.

Same tests when ran locally with newman 3.7.4 and node v7.4.0 pass without any issues.

**Comment by shankyty** [comment]
It seems to be same bug.

**Comment by shamasis** [comment]
@kunagpal - can we release latest newman docker and close this

**Comment by shankyty** [comment]
That would be a help our reporting is kind of screwed right now.

**Comment by kunagpal** [comment]
@shamasis This is blocked on #1105

**Comment by kunagpal** [comment]
@shankyty @klubi This bug is being fixed in #68
